apps/central-scan/frontend/src/components/export_results_modal.tsx

```diff
@@ -81,12 +81,9 @@ export function ExportResultsModal({ onClose }: Props): JSX.Element {
     if (usbDriveStatus === 'ejected') {
       return (
         <Modal
-          title="CVRs Saved"
+          title="USB Drive Ejected"
```

Comment:
I've made the CVR export modals in VxCentralScan and VxScan consistent. Prior, there were a number of miscellaneous inconsistencies both within and between the modals. I could of course set up a shared component but decided to punt on that for time's sake